adoc/deployment-loadbalancer.adoc

```diff
@@ -242,7 +242,7 @@ zypper in haproxy
 +
 [NOTE]
 ====
-Replace the individual `masterXX` with the IP/FQDN of your actual master nodes (one entry each) in the `server` lines.
+Replace the individual `<MASTER_XX>` with the IP/FQDN of your actual master nodes (one entry each) in the `server` lines.
```

Comment:
@r0ckarong what is meant  by "(one entry each)"?  - I'm not sure I understood if both the `<MASTER_00> <MASTER_00>` are supposed to get substituted for the IP/FQDN. Git blame says you wrote this line... :angel:

Reply:
For each master node that you want to use you must add another `server ...` line to the nginx.conf file. You replaced it correctly already.

Reply:
And only the second one needs to be substituted. The first one is the name the second one will be the URL under which that is reachable.

So `master00 master.example.com:6443` for instance.

Reply:
By URL you mean FQDN? 
Would this illustrate it more accurately? 
```server <MASTER_00_IP> <MASTER_00_FQDN>:6443 ```

Reply:
No I think the first part is an internal name in nginx. The second part must be replaced with EITHER the IP OR FQDN

Reply:
@lcavajani Can you clarify what the format for these entries in the config must be?

Reply:
Each entry corresponds to `server <name> <address>[:[port]] [param*]` ([documentation](https://cbonte.github.io/haproxy-dconv/1.7/configuration.html#server)). So the first argument is the internal `name` of the backend server, used for logging and alerts, whereas the second argument is the address of the backend server.